params/config.go

```diff
@@ -378,6 +381,19 @@ func (c *CliqueConfig) String() string {
 	return "clique"
 }
 
+// OptimismConfig is the optimism config.
+type OptimismConfig struct {
+	Enabled        bool           `json:"enabled"`
+	L1Block        common.Address `json:"l1_block"`
+	GasPriceOracle common.Address `json:"gas_price_oracle"`
+	FeeRecipient   common.Address `json:"fee_recipient"`
```

Comment:
Should we rename this to `BaseFeeRecipient`? And should we add a 2nd config var for the L1-cost fee recipient (and update state transition to direct the l1 cost fee there)? cc @trianglesphere

Reply:
Should probably do this (also FeeRecipient is overloaded at this point)

Reply:
Do we need a second recipient? If so, naming suggestions?

Reply:
Something like `BaseFeeRecipient` and `L1FeeRecipient`

Reply:
Done.